--- vpnc-0.3.3/vpnc-script~	2005-05-05 17:05:18.000000000 +0000
+++ vpnc-0.3.3/vpnc-script	2005-08-09 01:41:12.782263712 +0000
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #* reason                       -- why this script was called, one of: pre-init connect disconnect
 #* VPNGATEWAY                   -- vpn gateway address (always present)
 #* TUNDEV                       -- tunnel device (always present)
